terraform-provider-jans/jans/api_app_configuration.go

```diff
@@ -28,43 +28,47 @@ type AssetDirMapping struct {
 }
 
 type AssetMgtConfiguration struct {
-	AssetMgtEnabled          bool              `schema:"asset_mgt_enabled" json:"assetMgtEnabled"`
-	AssetServerUploadEnabled bool              `schema:"asset_server_upload_enabled" json:"assetServerUploadEnabled"`
-	AssetBaseDirectory       string            `schema:"asset_base_directory" json:"assetBaseDirectory"`
-	AssetDirMappings         []AssetDirMapping `schema:"asset_dir_mappings" json:"assetDirMappings"`
+	AssetMgtEnabled                bool              `schema:"asset_mgt_enabled" json:"assetMgtEnabled"`
+	AssetServerUploadEnabled       bool              `schema:"asset_server_upload_enabled" json:"assetServerUploadEnabled"`
+	FileExtensionValidationEnabled bool              `schema:"file_extension_validation_enabled" json:"fileExtensionValidationEnabled"`
+	ModuleNameValidationEnabled    bool              `schema:"module_name_validation_enabled" json:"moduleNameValidationEnabled"`
+	AssetBaseDirectory             string            `schema:"asset_base_directory" json:"assetBaseDirectory"`
+	JansServiceModule              []string          `schema:"jans_service_module" json:"jansServiceModule"`
+	AssetDirMappings               []AssetDirMapping `schema:"asset_dir_mappings" json:"assetDirMappings"`
```

Comment:
The `json:"assetDirMappings"` likely need to be rename to `json:"assetDirMapping"`? https://github.com/JanssenProject/jans/blob/e9e9808857fa58e70e02bd2fc054e7ae15b7200c/jans-linux-setup/jans_setup/templates/jans-config-api/dynamic-conf.json#L94